src/components/bs5/linkColumns/linkColumns.hbs

```diff
@@ -0,0 +1,13 @@
+{{#if data.items}}
```

Comment:
This is vanilla spec, unsure if we want to mimic? 

https://github.com/qld-gov-au/qgds-vanilla/blob/main/src/components/_global/css/link_columns/stories/link_columns.hbs

```
{{! linked_list.hbs v0.0.1 }}
{{#if component.data.items}}
    <ul{{#if component.data.metadata.id_field.value}} id="{{component.data.metadata.id_field.value}}"{{/if}} class="qld__link-columns{{#if component.data.type}} qld__link-columns--{{component.data.type}}{{/if}}">
    {{#each component.data.items}}
        <li  {{#if component.data.all_link }} class="qld__link-columns__all-link"{{/if}}>
          {{#if component.type }}
          {{> (lookup component "type") this }}
          {{else}}
          {{{ this }}}
          {{/if}}
      </li>
    {{/each}}
  </ul>
{{/if}}
```